INSTALL.txt

```diff
@@ -334,11 +334,12 @@ After a while I noticed, that bin/worker.php calls further php script via
 proc_open. These scripts themselves also use proc_open and fail, because they
 are NOT called with -d suhosin.executor.func.blacklist=none.
 
-So the simple solution is to put the correct parameters into config/local.ini.php:
+So the simple solution is to put the correct parameters into config/local.config.php:
 
-[config]
-; Location of PHP command line processor
-php_path = "/usr/bin/php -d suhosin.executor.func.blacklist=none -d suhosin.executor.eval.blacklist=none"
+	'config' => [
```

Comment:
For me this looks far more complicated then any system we have so far TBH ;-) The array syntax looks really confusing.

Reply:
I'm sorry about that, this is the price to have less support requests about special characters in the DB password.